.github/workflows/build-container.yaml

```diff
@@ -1,9 +1,6 @@
 name: Create, Test, and Publish Docker Image
 
-# Configures this workflow to run every time a change is pushed to the branch called `release`.
 on:
-  release:
-    types: [published]
```

Comment:
The "latest" tag runs whenever there is a tag pushed, which is a side-effect of the release given that we only ever tag and release at the same time.